cypress/support/commands.js

```diff
@@ -1,22 +0,0 @@
-import waitForDom from "./waitForDom"
```

Comment:
Just curious, does cypress 10 no longer need waitForDom to reduce test flakiness?

Reply:
oof i missed this. the `waitForDom` actually didnt seem to have any effect - this might be because i added some assertions. 

just double checking here - the `waitForDom` prevents the `xxx is detached from the dom` error? if so, i think it might be useful for me to undo the deletion and instead, migrate it to ts

Reply:
Yeah this seemed to help with the issue originally - not sure if moving to cypress 10 helps already though. If the tests are less flaky now, maybe it doesn't need to be readded?